lib/api/CDataFrameAnalysisInstrumentation.cc

```diff
@@ -153,6 +162,18 @@ void CDataFrameAnalysisInstrumentation::monitorProgress(
             lastProgress = progress;
             writeProgress(lastTask, lastProgress, &writer);
         }
+        LOG_DEBUG(<< "Memory usage " << instrumentation.memory());
```

Comment:
This will generate quite a lot of noise in unit tests, I'd suggest downgrading to LOG_TRACE.
```suggestion
        LOG_TRACE(<< "Memory usage " << instrumentation.memory());
```

Reply:
I removed the trace completely. IMO it has limited benefit and produces a lot of noise.